client/mksubvolume.cc

```diff
@@ -258,6 +277,33 @@ find_filesystem(MntTable& mnt_table)
     }
 }
 
+struct TmpMountpoint {
```

Comment:
I suggest to make mountpoint and fs private and add a get_path() function.

Reply:
Done.